helper/resource/testing_new_import_state.go

```diff
@@ -128,25 +126,16 @@ func testStepNewImportState(ctx context.Context, t testing.T, helper *plugintest
 			importConfig = appendImportBlock(importConfig, resourceName, importId)
 		}
 
-		confRequest := teststep.PrepareConfigurationRequest{
-			Directory: step.ConfigDirectory,
```

Comment:
We already know that ConfigDirectory and ConfigFile are zero-valued at this point.